### PR TITLE
s3 instead of sciencebase

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
If we stick with this pattern, we could conceivably want to make an s3 fetcher for `vizlab`?

File: scripts/fetch/fetchGA.R [hunk 1]
```diff
@@ -3,21 +3,25 @@ library(googleAnalyticsR)
...
+    print(sessionInfo())
     #get file from sb with fetcher
-    viz[['fetcher']] <- 'sciencebase'
-    viz[['remoteFilename']] <- basename(viz[['location']])
-    message("Downloading SB file...")
-    fetch(as.fetcher(viz))
-    message('Downloaded SB file')
-
+    message("Downloading from S3")
+    use_credentials(profile = "default")
```
Comment (jiwalker-usgs) [inline scripts/fetch/fetchGA.R]: I think we would want the profile to be read in from some environment variable (we could isolate the tiers this way)

File: scripts/fetch/fetchGA.R [hunk 2]
```diff
@@ -3,21 +3,25 @@ library(googleAnalyticsR)
 library(googleAuthR)
 library(dplyr)
 library(data.table)
-library(sbtools)
 library(assertthat)
+library(aws.s3)
+library(aws.signature)
```
Comment (jiwalker-usgs) [inline scripts/fetch/fetchGA.R]: Does this need to be in the viz.yaml, or is counting on transitive dependencies in this case good enough?

Comment (wdwatkins) [inline scripts/fetch/fetchGA.R]: Yes, I will add it, the `use_credentials` function is from `aws.signature`

**Pull request merged**